src/appengine/handlers/base_handler.py

```diff
@@ -300,3 +304,160 @@ def get_upload(self):
 
     self.upload = blob_info
     return self.upload
+
+
+class FlaskHandler(MethodView):
```

Comment:
Can we put flask specific stuff in its own module, rather than having both webapp2 and flask related stuff in the same one.

Reply:
done